packages/analytics/amplify_analytics_pinpoint_dart/test/queued_item_store_test.dart

```diff
@@ -0,0 +1,195 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/dart_queued_item_store.dart';
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/queued_item_store.dart';
+
+import 'package:test/test.dart';
+
+void main() {
+  late DartQueuedItemStore db;
+
+  group('DartQueuedItemStore ', () {
+    setUpAll(() {
+      db = DartQueuedItemStore('/tmp');
```

Comment:
```suggestion
      final tempDir = Directory.systemTemp.createTempSync('amplify_analytics');
      addTearDown(() => tempDir.deleteSync(recursive: true));
      db = DartQueuedItemStore(tempDir.path);
```

Reply:
I don't think we can do that as Directory is from `dart:io`

Reply:
Sure you can.

```dart
String? path;
if (!zIsWeb) {
  final tempDir = Directory.systemTemp.createTempSync('amplify_analytics');
  addTearDown(() => tempDir.deleteSync(recursive: true));
  path = tempDir.path;
}
db = DartQueuedItemStore(path);
```

Reply:
Nice thanks!